### PR TITLE
Dev migrate to diglib rails

### PR DESCRIPTION
**Pull request by nwalker2398**
Deployed to `dev`, `test`, and `prod` on diglib-rails servers. Verified working order of each in-browser using etc/hosts local override.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -11,6 +11,9 @@
 /db/*.sqlite3
 /db/*.sqlite3-*
 
+# Ignore OS-specefic hidden files.
+.DS_Store
```
Comment (elohanlon) [inline .gitignore:15]: This is good to have here!  I'd also recommend adding a local environment config file like `~/.gitignore_global` with this content:

```
*~
\#*\#
.DS_Store
```

And then you can run this command to use that file as a global ignore list:

`git config --global core.excludesFile '~/.gitignore_global'`

That way, if you're ever working with a repository where someone else forgot to add .DS_Store to the repo's .gitignore, you can avoid accidentally committing .DS_Store (and other globally ignored files) to it.

File: config/database.template.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+development:
```
Comment (elohanlon) [inline config/database.template.yml:1]: It looks like this file is a duplicate of `config/templatest/database.template.yml`.  Okay to remove?

**Review by elohanlon — CHANGES_REQUESTED** [review]
I just had one minor comment about a duplicate file, but otherwise looks good!

File: config/database.template.yml (hunk 2 above)
Comment (nwalker2398) [reply config/database.template.yml:1]: Absolutely! Good catch!

**Review by elohanlon — APPROVED** [review]

**Pull request merged**